src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography.Tests/Microsoft.Azure.KeyVault.Cryptography.Tests.csproj

```diff
@@ -5,7 +5,7 @@
     <Description>Microsoft Azure Key Vault Cryptography tests</Description>
     <AssemblyTitle>Microsoft Azure Key Vault Cryptography tests</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.KeyVault.Cryptography.Tests</AssemblyName>
-    <VersionPrefix>3.0.0-alpha</VersionPrefix>    
+    <VersionPrefix>3.0.1</VersionPrefix>    
```

Comment:
Please replace `VersionPrefix` with `Version`
Test projects are never publish, please set the version to `1.0.0` and leave it unchanged